website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -155,7 +155,60 @@ When `format: csv`, can either supply:
 
 ### Overrides
 
-When configuring your unit test, you can override the output of macros, vars, or environment variables for a given unit test. 
+When configuring your unit test, you can override the output of [macros](https://docs.getdbt.com/docs/build/jinja-macros#macros), [project variables](https://docs.getdbt.com/docs/build/project-variables), or [environment variables](https://docs.getdbt.com/docs/build/environment-variables) for a given unit test. 
```

Comment:
I'm not sure if these are the correct adjustments or not, but the style guide calls for removing hard-code `https://docs.getdbt.com` in favor of relative references.

```suggestion
When configuring your unit test, you can override the output of [macros](/docs/build/jinja-macros#macros), [project variables](/docs/build/project-variables), or [environment variables](/docs/build/environment-variables) for a given unit test. 
```

Reply:
Yep that's correct, @dbeatty10!

Reply:
Thanks for catching that!